.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 matrix:
   include:
-    - node_js: 10
+    - node_js: 12
```

Comment:
Once this has been updated to use the more simple Travis config, I can give +1

Reply:
Updated.